### DISCUSSION TITLE
Integration not found suddenly

### DISCUSSION THREAD

**Discussion by vstarcruzer** [sebr/bhyve-home-assistant#126] [Q&A]
So I have installed this integration through HACS about a week ago. Everything was working fine than, but suddenly I lost my attritubes to most switches and sensors and now I cannot restart HA because "Integration "bhyve" is not found". Any ideas on if there was a recent issue with HA and this integration? I tried removing it and then reinstalling it through HACS but unless I comment out the configuration, sensor, switch, and binary switch for bhyve, my HA will not restart. I am at a loss.

**Comment by sebr** [comment]
No idea... check the 	HA logs and also file/directory permissions.

**Comment by vstarcruzer** [comment]
I figured it out, it had to update to 2.0.6. Now it says it has no module
for binary_sensor. I noticed it didn't have the binary_sensor.py in custom
components so I copied it there. Still HA still says it can't find it, but
I do have switches and sensors working again.

**Reply by bhaywa1** [reply]
I'm having a similar issue today all of a sudden. HASS was having issues with running through a local switch, so I added a different switch. Same issues, so pulled the switch out completely. However, after rebooting HASS, I now get several errors...starting with "The b-hyve integration is depreciated..." (see below). 
![Untitled](https://user-images.githubusercontent.com/79614768/184020911-abffaf58-38ba-4083-9200-e97a4b4079e0.png)

Wondering if I need to reinstall the b-hyve integration or if something else is going on... I'm not able to access any of the switches, devices, etc.

Thoughts anyone? The system was working just fine for two or three weeks. 

Thanks!

**Reply by bhaywa1** [reply]
BTW, i'm on v3.0.0. Does not appear to be any more updates at this time. And it worked just fine initially.
